kuma/javascript/src/newsletter.jsx

```diff
@@ -221,15 +235,14 @@ export default function Newsletter() {
                             ref={errorsRef}
                             id="newsletter-errors"
                             className={
-                                errorsArray ? 'newsletter-errors' : 'hidden'
+                                hasErrors ? 'newsletter-errors' : 'hidden'
```

Comment:
The "React way" is to not use CSS to show or hide things. I think I would do it like this:

```jsx
{hasErrors && (<div id="newsletter-errors" className="newsletter-errors">
    <ul className="errorlist">
   ...
</div>)}
```

By the way, I see a lot of redundancy. Why do we need both an ID and a class? And can't you get to the `ul` with CSS simply by doing `.newsletter-errors ul`?

Reply:
True, also prefer the React way and hence changed it just now. Didn't touch the CSS so far because I didnt feel strongly about it yet